### DISCUSSION TITLE
Make extension available as an embed for the website

### DISCUSSION THREAD

**Discussion by pjeweb** [alveusgg/extension#23] [Ideas]
Would it be helpful for the Alveus team if the extension could be used on the website?

I would assume right now this is redundant work to update both the wordpress pages and the extension. Currently the website has some more pictures than the extension but those could be added to the extension as well (see #19)

To integrated it into Wordpress the easiest way would be to host the view on github pages or something like that and iframe it into the website  (similar to the panel view), but that would loose the deep links (URL) for the ambassadors, so maybe it would have to be an actual wordpress plugin and some rewrite rule [1] to support deep links.

1) https://developer.wordpress.org/reference/functions/add_rewrite_rule/

**Comment by abdullahmorrison** [comment]
ye it would be redundant, and it would just add clutter to the site.

**Reply by pjeweb** [reply]
I more so thought we could provide a different design more suited to the website but using the same data just so any updates and features you add to the extension can also be used on the website. They would share some components but most importantly the base data. All of that could be seamlessly integrated with a borderless iframe. If done well it should not be cluttered at all. In my mind this would replace the content on the existing ambassador pages ([1] [2] etc.) and would not be embedded on different pages.

1) https://www.alveussanctuary.org/ambassadors/
2) https://www.alveussanctuary.org/ambassadors/georgie/
